CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## Unreleased
+
+### Fixed
+- `filter-dropdown-remove:after` no longer positioned absolutely
+
+### Modified
+- `btn-clear` has bg white, rathe than transparent for ACAG on light grays.
```

Comment:
@nikolaswise @paulcpederson  shouldn't we have kept the original btn-clear for cases where having a Transparent background is fine. and introduced a new button style class with the same border and hover as btn-clear but with an additional white fill/background color?